### ISSUE TITLE
Auto reconnecting after the round end is bugged

### ISSUE THREAD

**Issue by Accinator50** [ParadiseSS13/Paradise#24328]
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1630

### Issue Description

The game doesn't properly reconnect after the round ended. 

I've also asked other people on the discord and they had the same problem.

### What did you expect to happen?

It should properly reconnect.

### What happened instead?

I stole this picture from somebody on the discord. 
![grafik](https://github.com/ParadiseSS13/Paradise/assets/44909173/c607e689-c597-4bea-8b42-bdc0fd799e8f)

### Why is this bad/What are the consequences?

It's an annoying bug.

### Steps to reproduce the issue.

Wait for the auto reconnect after the round ended.

### When did the problem start happening?

This has been the case for me since TGUI was merged, but it got worse with the TM of #24065.

Before it just didn't reconnect and you needed to do it manually. Now you need to close each instance and log into paradise from the Byond hub.

**Comment by S34NW** [comment]
Not an auto reconnect bug, happens on first join every now and then too

**Comment by S34NW** [comment]
This might be a BYOND issue, it's always happened to some degree

**Comment by Accinator50** [comment]
> This might be a BYOND issue, it's always happened to some degree

I'm honestly not sure if these two problems (not being able to log in and the game not automatically reconnecting) are the same two bugs, as the first one has been around for a very long time, but I honestly never had a problem with the automatic reconnection until recently, and as mentioned above, the problem for me is that it stopped working at all. The game always crashes. That's why I decided to write this issue report. However, it's possible that Byond recently released an update that crashes the game (I updated the game after the older version I was using prevented me from joining Paradise - I didn't really notice the problem back then but maybe that was because I didn't play a lot of rounds back when I updated it).
